src/utils/ClientConfig.ts

```diff
@@ -0,0 +1,26 @@
+import { Logger } from "@aws-lambda-powertools/logger";
+import { MessageCodes } from "../models/enums/MessageCodes";
+import { HttpCodesEnum } from "./HttpCodesEnum";
+import { AppError } from "./AppError";
+
+type ClientConfig = {
+	jwksEndpoint: string;
+	clientId: string;
+	redirectUri: string;
+	YotiBaseUrl: string;
+	GovNotifyApi: string;
+};
+
+export function getClientConfig(clientConfig: string, sessionClientId: string, logger: Logger): ClientConfig | undefined {
```

Comment:
Please can we have some unit tests for this?

Reply:
Added.